activation/activation.go

```diff
@@ -440,17 +419,17 @@ func (b *Builder) PublishActivationTx() error {
 	defer b.db.UnsubscribeAtx(atx.ID())
 	size, err := b.signAndBroadcast(atx)
 	if err != nil {
-		b.log.With().Error("failed to publish atx", append(atx.Fields(b.layersPerEpoch, size), log.Err(err))...)
+		b.log.With().Error("failed to publish atx", append(atx.Fields(size), log.Err(err))...)
 		return err
 	}
 
-	b.log.Event().Info("atx published!", atx.Fields(b.layersPerEpoch, size)...)
+	b.log.Event().Info("atx published!", atx.Fields(size)...)
 	events.Publish(events.AtxCreated{Created: true, ID: atx.ShortString(), Layer: uint64(b.currentEpoch())})
 
 	select {
 	case <-atxReceived:
 		b.log.Info("atx received in db")
-	case <-b.layerClock.AwaitLayer((atx.TargetEpoch(b.layersPerEpoch) + 1).FirstLayer(b.layersPerEpoch)):
+	case <-b.layerClock.AwaitLayer((atx.TargetEpoch() + 1).FirstLayer(b.layersPerEpoch)):
```

Comment:
you could probably do with `FirstLayer` what you did with `TargetEpoch` for consistency